office365.json

```diff
@@ -2825,6 +2839,121 @@
             },
             "versions": "EQ(*)"
         },
+        {
+            "action": "delete event",
+            "identifier": "delete_event",
+            "description": "Delete an event from user calendar",
+            "type": "generic",
+            "read_only": false,
+            "parameters": {
+                "id": {
+                    "description": "Event ID to delete",
+                    "data_type": "string",
+                    "required": true,
+                    "primary": true,
+                    "contains": [
+                        "msgoffice365 event id"
+                    ],
+                    "order": 0
+                },
+                "email_address": {
+                    "description": "Email address of the mailbox owner",
+                    "data_type": "string",
+                    "required": true,
+                    "primary": true,
+                    "contains": [
+                        "email"
+                    ],
+                    "order": 1
+                },
+                "send_response": {
```

Comment:
should this be called `send_decline_response`?


`send_response` is quite generic, as response can be anything.

Reply:
updated